src/pam_raw.rs

```diff
@@ -174,6 +174,18 @@ impl fmt::Display for PamError {
     }
 }
 
+pub fn get_user(pamh: PamHandle, prompt: Option<*const c_char>) -> PamResult<Option<*const c_void>> {
```

Comment:
Ah you noticed :) I did that because the rest of the items are all *void, so I can handle it in the same way. Returning a *c_char instead means a few extra lines of code. No prob.